docs/source/tracking.rst

```diff
@@ -543,12 +547,25 @@ along with its scheme and port (for example, ``http://10.0.0.1:5000``) or call :
 The :py:func:`mlflow.start_run`, :py:func:`mlflow.log_param`, and :py:func:`mlflow.log_metric` calls 
 then make API requests to your remote tracking server.
 
-.. code-block:: py
+  .. code-section::
+
+    .. code-block:: python
+
+        import mlflow
+        remote_server_uri = "..." # set to your server URI
+        mlflow.set_tracking_uri(remote_server_uri)
```

Comment:
You need to set the experiment first, since there's no default experiment. We should definitely include instructions (like you need to have a legit workspace path as well)